tests/dogtag/pytest-ansible/pytest/ca/bugzilla/test_bug_1787115_Copy_SKI_from_CSR_to_Certificate_Signed.py

```diff
@@ -0,0 +1,258 @@
+	from pki.testlib.common import constants
+except Exception as e:
+	if os.path.isfile('/tmp/test_dir/constants.py'):
+		sys.path.append('/tmp/test_dir/')
+		import constants
+
+profile = 'caCACert'
+profop = ProfileOperations(nssdb=constants.NSSDB)
+
+
+def test_profile_update_SKI_parameters(ansible_module):
+	"""
+		:id: 8f761dd1-219e-404a-a1ea-333cf2a606b9
+		:Title: Bug 1787115 - SubjectAltNameExtInput does not display text fields to the enrollment page
+		:Description: Bug 1787115 - SubjectAltNameExtInput does not display text fields to the enrollment page
+		:Requirement:
+		:CaseComponent: \-
+		:Setup:
+			1. Use the subsystems setup in ansible to run subsystem commands
+
+		:Steps:
+			1. Disable profile caCACert.
+			2. Edit caCACert profile to add parameter 'policyset.caCertSet.8.default.params.useSKIFromCertRequest=true'
+			3. Enable profile.
+
+		:ExpectedResults:
+			1. profile should be disabled.
+			2. caCACert profile must include parameter 'policyset.caCertSet.8.default.params.useSKIFromCertRequest=true'
+			3. profile should be enabled.
+
+	"""
+	tmp_file = tempfile.mktemp(suffix='tmp_', prefix='_profile')
+	add_param = {'policyset.caCertSet.8.default.params.useSKIFromCertRequest': 'true'}
+	profile_show = ansible_module.pki(cli='ca-profile-show',
+	                                  nssdb=constants.NSSDB,
+	                                  port=constants.CA_HTTP_PORT,
+	                                  dbpassword=constants.CLIENT_DATABASE_PASSWORD,
+	                                  certnick='"{}"'.format(constants.CA_ADMIN_NICK),
+	                                  extra_args='{} --raw'.format(profile))
+	
+	for _, result in profile_show.items():
+		if result['rc'] == 0:
+			newProfile = result['stdout']
+			if newProfile:
+				with open(tmp_file, "a+") as f:
+					f.write(newProfile + '\n')
+					for key, value in add_param.items():
+						f.write(str(key) + '=' + str(value) + '\n')
+					f.close
+			else:
+				log.error("Failed to get caServerCert Profile")
+				log.error("Failed to run: {}".format(result['cmd']))
+				log.error(result['stdout'])
+				log.error(result['stderr'])
+				pytest.fail()
+	
+	profop.disable_profile(ansible_module, profile)
+	
+	ansible_module.copy(src=tmp_file, dest='/tmp/caCACert.tmp')
+	
+	update_prof = ansible_module.pki(cli='ca-profile-mod',
+	                                 nssdb=constants.NSSDB,
+	                                 port=constants.CA_HTTP_PORT,
+	                                 dbpassword=constants.CLIENT_DATABASE_PASSWORD,
+	                                 certnick='"{}"'.format(constants.CA_ADMIN_NICK),
+	                                 extra_args='/tmp/caCACert.tmp --raw')
+	for result in update_prof.values():
+		if result['rc'] == 0:
+			log.info('Profile Changed Successfully.')
+		else:
+			log.error("Failed to Update Profile.")
+			pytest.fail("Failed to Update Profile.")
+	
+	enabled = profop.enable_profile(ansible_module, profile_name=profile)
+	assert enabled
```

Comment:
Can remove assert enabled, i do't think it's doing anything here.

Reply:
I think the method returns True. And assert validates if the profile is enabled or not.!

Reply:
Not sure it would return value in boolean, if value return is other than boolean it would given same result every time.